config/default.yml

```diff
@@ -3491,10 +3491,12 @@ Style/EmptyElse:
   # empty - warn only on empty `else`
   # nil - warn on `else` with nil in it
   # both - warn on empty `else` and `else` with `nil` in it
+  # comment - same as `both`, but if there is a comment in `else`, it's acceptable
   SupportedStyles:
     - empty
     - nil
     - both
+    - comment
```

Comment:
Perhaps `allow_comment` would be a better name here? Thinking more about this - probably this shouldn't be a style but rather an option `AllowComments` that affects all styles.

Reply:
(false by default)

Reply:
Thank you very much. I changed to the `AllowComments` option. I think this is better too.